src/ui_fsmenu/launch_mpg.cc

```diff
@@ -329,9 +248,10 @@ void FullscreenMenuLaunchMPG::select_saved_game() {
 		Section& s = prof.get_safe_section("global");
 
 		std::string mapname = s.get_safe_string("name");
-		nr_players_ = s.get_safe_int("nr_players");
+		auto nr_players_ = s.get_safe_int("nr_players");
```

Comment:
```suggestion
		const int nr_players = s.get_safe_int("nr_players");
```
don't use `auto` for primitive types, and no trailing underscore in local variable names please

Reply:
done